### DISCUSSION TITLE
Governance update

### DISCUSSION THREAD

**Discussion by julian-klode** [julian-klode/dns66#480] [Announcements]
I'm sorry to say, but I have stopped using DNS66. I'm happy to accept patches, cut releases, and provide stewardship to ensure it continues to provide the level of privacy that was promised, but I do not generally have the capacity to undertake development myself or interact with bug reports.

One aspect where DNS66 lacks in comparison to DoT-based ad blocking is reliability and power consumption. That was ultimately much of a driver for me to switch to DoT, as it "just works".

I am concerned that alternatives do not provide the same level of privacy as DNS66. While DNS66 is not as featureful as others, it always had a very strict policy regarding privacy.

**Comment by Dragoon91786** [comment]
# I Bow To Your Awesome! 😎

Thank you for the project! I have found it _**EXTREMELY RELEVANT & HELPFUL**_ protecting my internet privacy/security.

Thank you GREATLY for all of your personage sacrifices and work on this project (and to all those that assisted)!

For those of us unable to circumvent Google through various means, this app has been truly wonderful and kicked the crap out of so much tracking-ad trash! ❤️💯

# Miscellaneous Query—Documentation

I had a general question regarding integrating my I-Blocklist blacklists and whitelists. What archive format or format should I be using for the URL?

I included some screenshots for comparison since there were not any documentation related to this aspect of the app. Pic one is the general layout of the site. Pic 2 is of File Format for Blocklist. Pic 3 is of Archive formats. Wondered if you might be able to shed some light on this? Would be not only helpful to me, but also to others possibly wanting to pickup the app in the future. ❤️💯🙏🏽🙇🏽



![Screenshot_20220905-232814.png](https://user-images.githubusercontent.com/48355906/188562900-daa958f1-4b63-471e-9b07-a04eee44a757.png)

![Screenshot_20220905-232823.png](https://user-images.githubusercontent.com/48355906/188562915-b3798f5c-267b-48b0-a761-c1d5d56e391c.png)

![Screenshot_20220905-232829.png](https://user-images.githubusercontent.com/48355906/188562937-de14e15e-753b-42cd-8925-48d88d4c66db.png)

As always, COPIOUS understanding and thanks for what you have done and what you do!

**Reply by julian-klode** [reply]
You want uncompressed hosts files. Basically each line contains `127.0.0.1 hostname` or `0.0.0.0 hostname` or I think just hostnames or `::1 hostname` also work, but not sure.

**Comment by paulaime** [comment]
Thanks for your great work on this, I've been using it for many years. What do you recommend instead of DNS66 then? Blokada?

**Reply by julian-klode** [reply]
I don't think it needs something else, just someone has to contribute patches to keep things working.

As I said before, I do not trust other apps. They open a VPN, so they can capture all network traffic.

Personally I mostly use a DNSoverTLS service. I've used Adguard DNS and am currently playing around with RethinkDNS. With RethinkDNS, you can configure custom blocklists. Quite fancy, a server-side DNS66 if you want to call it that. You can even "self-host" it, albeit it runs on like cloudflare workers so you make of that what you want.

**Reply by ignoramous** [reply]
Julian, I am mostly humbled and surprised you use Rethink! (:

Btw, `max.rethinkdns.com` runs on Fly.io (default on F-Droid and GitHub versions), `rdns.deno.dev` (alpha quality) runs on GCP (not used because expensive), and `sky.rethinkdns.com` runs on Cloudflare (default on Play Store, but we've been recently banned from there). From the android app, there's a way to switch between Max and Sky. I'd imagine Sky is faster because Cloudflare.

**Reply by julian-klode** [reply]
I'm using the Private DNS toggle Widget from f-droid with an address generated from the web frontend which makes my life easy and avoids any VPN mechanism worrying.

I think the app still needs a VPN tunnel, as the API for private DNS toggling is only available with special permissions.

**Reply by julian-klode** [reply]
I was using the widget with Adguard DNS for a while but to clearly Rethink DNS is better.

And with the native Android private DNS used, the widget adds the value of being able to toggle it off to unblock a specific site.

**Reply by ignoramous** [reply]
Gotcha. I am surprised Rethink DNS performs or is in any shape better than Adguard DNS. Most certainly not as feature-rich as NextDNS. We've come a long way, I guess.

Re: app: Yeah, it needs the VPN tunnel, but we intend to add support for users to connect to any upstream Wireguard server by Jan 2023 :crossed_fingers:  The advantage with the app is, you can see the logs in real-time on-device (battery may take a hit because of the IP Firewall, but if one only needs DNS, then there's a DNS-only mode in the app). And soon-ish, we'd add ability for users to allowlist individual domains and setup blocklists and allowlists *per-app* (rather than device-wide). Something we have been working on for about 3 months now. Let's see how it goes as we try to wrap up the development.

**Comment by shuvashish76** [comment]
https://f-droid.org/en/packages/dev.clombardo.dnsnet fork that _aims to continue the goals of the original app with modern Android development practices_.

**Reply by matte78** [reply]
it is always open source with a git repo?

**Reply by shuvashish76** [reply]
Yes https://github.com/t895/DNSNet
All the apps on F-Droid are Free and Open Source Software.
